tensorflow/lite/delegates/gpu/gl/kernels/add.cc

```diff
@@ -35,116 +35,128 @@ namespace {
 
 class Add : public NodeShader {
  public:
-  absl::Status GenerateCode(const GenerationContext& ctx,
-                            GeneratedCode* generated_code) const final {
-    const auto& attr =
-        absl::any_cast<const ElementwiseAttributes&>(ctx.op_attr);
-    auto adds = absl::get_if<Tensor<Linear, DataType::FLOAT32>>(&attr.param);
-    auto scalar = absl::get_if<float>(&attr.param);
-
-    const auto* hwc_tensor =
-        absl::get_if<Tensor<HWC, DataType::FLOAT32>>(&attr.param);
-
-    if (!adds && !scalar && !hwc_tensor) {
-      // check if it is a broadcast
-      if (ctx.input_shapes.size() == 2 &&
-          ctx.input_shapes[0] != ctx.input_shapes[1] &&
-          ctx.input_shapes[1][1] == 1 && ctx.input_shapes[1][2] == 1 &&
-          ctx.input_shapes[0][3] == ctx.input_shapes[1][3]) {
-        // TODO(b/147771327): investigate why input_data_1[gid.z] worked before
+    absl::Status GenerateCode(const GenerationContext& ctx,
+        GeneratedCode* generated_code) const final {
+        const auto& attr =
+            absl::any_cast<const ElementwiseAttributes&>(ctx.op_attr);
+        auto adds =
+            absl::get_if<Tensor<Linear, DataType::FLOAT32>>(&attr.param);
+        auto scalar = absl::get_if<float>(&attr.param);
+
+        const auto* hwc_tensor =
+            absl::get_if<Tensor<HWC, DataType::FLOAT32>>(&attr.param);
+
+        if (hwc_tensor) {
+            *generated_code = {
+                /*parameters=*/{},
+                /*objects=*/
+                {{"hwc_buffer",
+                MakeReadonlyObject(
+                    uint3(static_cast<int>(ctx.input_shapes[0][2]),
+                        static_cast<int>(ctx.input_shapes[0][1]),
```

Comment:
oh, no.  you misunderstood my comment.  I didn't ask the entire file to be indented by 4, but to indent 

*generated_code = {
// indent by 4 here instead of just 2
};

The rest of the code should be just indent by 2.  Usually, clang-format -style Google will be able to deal with all this.

Reply:
ups